templates/header.mustache

```diff
@@ -17,44 +17,68 @@
-        </div>
-
-        <a href="{{{ config.wwwroot }}}" class="navbar-brand {{# output.should_display_navbar_logo }}has-logo{{/ output.should_display_navbar_logo }}
-            {{^ output.should_display_navbar_logo }}
-                hidden-sm-down
-            {{/ output.should_display_navbar_logo }}
-                ">
-            {{# output.should_display_navbar_logo }}
-                <span class="logo hidden-xs-down">
-                    <img src="{{output.get_compact_logo_url}}" alt="{{sitename}}">
-                </span>
-            {{/ output.should_display_navbar_logo }}
-            <span class="site-name hidden-sm-down">{{{ sitename }}}</span>
-        </a>
-
-        <!-- user_menu -->
-        {{{ output.user_menu }}}
-
-        <!-- navbar_plugin_output -->
-        {{{ output.navbar_plugin_output }}}
-
-        <!-- search_box -->
-        <span class="hidden-md-down">
-        {{{ output.search_box }}}
-        </span>
-
-        <nav class="nav navbar-nav hidden-md-down">
-            <!-- custom_menu -->
-            {{{ output.custom_menu }}}
-            <!-- page_heading_menu -->
-            {{{ output.page_heading_menu }}}
-        </nav>
-    </div>
+{{{ ribbonhtml }}}
+{{{ partnershipinfo }}}
+<header role="banner" class="c-page-header" id="jsPageHeader">
+
+    <div class="o-wrapper  c-page-header__inner">
+
+          <a href="{{{config.wwwroot}}}/my" class="c-page-header__lockup">
+            <img src="{{{ logosrc }}}" alt="NHS" width="204" height="62" class="c-page-header__logo">
+          </a>
+
+           <div class="c-page-header__controls">
+
+              <button class="c-nav-trigger" id="jsNavTrigger" aria-label="menu" data-expands="#jsNav" role="button" aria-expanded="false">☰</button>
+
+              <nav class="c-nav-primary  c-page-header__nav" id="primaryNav">
+
+                <ul class="c-nav-primary__list" id="jsNav">
+
+                  <li class="c-nav-primary__item c-nav-primary__align">
+                    <a href="{{{config.wwwroot}}}/my" class="c-nav-primary__link">Dashboard</a>
+                  </li>
+
+                  <li class="c-nav-primary__item c-nav-primary__align">
+                    <button class="c-nav-primary__link" data-expands="#item-2-sub" data-popup="" role="button" aria-expanded="false" aria-haspopup="true">Programmes</button>
+                        <nav class="c-nav-primary__sub  jsNavSub" id="item-2-sub" role="group" aria-label="sub navigation menu">
+
+                            <div class="o-wrapper">
+
+                                <div class="o-layout">
+                                    <div class="o-layout__item  u-6/12@lg">
+
+                                                <ul class="c-nav-primary__sub-links">
+                                                    <li>
+                                                        {{{availablecourses}}}
```

Comment:
If there are no available courses does this leave us with empty markup or does it return a "no courses" message?

Reply:
@cehwitham  No courses check has been added. Pls review the latest commit below.